apps/glusterfs/app_volume.go

```diff
@@ -246,8 +246,22 @@ func (a *App) VolumeDelete(w http.ResponseWriter, r *http.Request) {
 			return err
 		}
 
-		return nil
+		if !volume.Info.Block {
```

Comment:
@lpabon this is all in the end-to-end design document proposed under gluster-kubernetes: https://github.com/gluster/gluster-kubernetes/pull/268